advanced/hpc_compute_to_data/jupyter_notebook/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM jupyter/base-notebook
 
-ARG  irods_gid=999
+ARG  irods_uid=997
```

Comment:
I can assume that the default values given here apply within the approved(TM) sequence of presentations for UGM 2023 Advanced Training...? Not that it will matter, I guess - since we'll be changing the slides to use the --build-var's

Reply:
I just changed it to what my values were, but yeah, we will build it as described in the commit message for training, so it shouldn't matter, in theory.

Reply:
Going to resolve this as I have confirmed that it works on a fresh VM instance.